pyiron_base/generic/flattenedstorage.py

```diff
@@ -18,7 +18,9 @@
 __date__ = "Jul 16, 2020"
 
 
+import chunk
 import copy
+from multiprocessing.sharedctypes import Value
```

Comment:
```suggestion
import copy
```
I suppose this is left over from something else.

Reply:
Maybe some were autoimports from some pseudo-smart code extension